astropy/wcs/tests/extension/test_extension.py

```diff
@@ -30,10 +32,18 @@ def test_wcsapi_extension(tmpdir):
     # *unless* the output is redirected.  This bug also did not occur in an
     # interactive session, so it likely had something to do with pytest's
     # output capture
+    #
+    # --single-version-externally-managed --record is essentially equivalent to
+    # what pip does; otherwise setuptools' `setup.py install` will try to build
+    # and install an egg (deprecated behavior, but still the default for
+    # backwards-compat).
+    # In the future we might change this to just run pip.
```

Comment:
Should open an issue so we don't forget.

Reply:
OK, done: #11901.   Doing any more work on this is not so high priority, but at least now there's a reminder.